hardhat.config.ts

```diff
@@ -0,0 +1,84 @@
+import { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-helpers"
+import "@keep-network/hardhat-local-networks-config"
+import "@nomiclabs/hardhat-waffle"
+import "hardhat-gas-reporter"
+import "hardhat-deploy"
+import "@tenderly/hardhat-tenderly"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.4",
+      },
+    ],
+  },
+  paths: {
+    artifacts: "./build",
+  },
+  networks: {
+    hardhat: {
+      forking: {
+        // forking is enabled only if FORKING_URL env is provided
+        enabled: !!process.env.FORKING_URL,
+        // URL should point to a node with archival data (Alchemy recommended)
+        url: process.env.FORKING_URL || "",
+        // latest block is taken if FORKING_BLOCK env is not provided
+        blockNumber: process.env.FORKING_BLOCK
+          ? parseInt(process.env.FORKING_BLOCK)
+          : undefined,
+      },
+      tags: ["local"],
+    },
+    development: {
+      url: "http://localhost:8545",
+      chainId: 1101,
+      tags: ["local"],
+    },
+    ropsten: {
+      url: process.env.CHAIN_API_URL || "",
+      chainId: 3,
+      accounts: process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY
+        ? [process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY]
+        : undefined,
+      tags: ["tenderly"],
+    },
+  },
+  tenderly: {
+    username: "thesis",
+    project: "",
```

Comment:
The hardhat tenderly plugin doesn't work perfectly yet (https://github.com/Tenderly/hardhat-tenderly/issues/20), but let's define the project slug we created in Tenderly to be used here.
```suggestion
    project: "thesis/threshold-network",
```

Reply:
Added, see 32232130fe1b975f19ebf10764bae6ea9632afd6.